Assets/MixedRealityToolkit/_Core/EventDatum/Input/MixedRealityPointerEventData.cs

```diff
@@ -31,10 +31,20 @@ public class MixedRealityPointerEventData : InputEventData
         /// </summary>
         /// <param name="pointer"></param>
         /// <param name="inputAction"></param>
+        /// <param name="handedness"></param>
+        /// <param name="inputSource"></param>
         /// <param name="count"></param>
-        public void Initialize(IMixedRealityPointer pointer, MixedRealityInputAction inputAction, int count = 0)
+        public void Initialize(IMixedRealityPointer pointer, MixedRealityInputAction inputAction, Handedness handedness = Handedness.None, IMixedRealityInputSource inputSource = null, int count = 0)
```

Comment:
again pointers shouldn't know about handedness, although I do see the benefit of the input source reference.

Reply:
Pointers [already know about handedness](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit/_Core/EventDatum/Input/MixedRealityPointerEventData.cs#L49).

Reply:
No. They don't have a property with handedness. Nor should they. Those kinds of properties only exist in the IMixedRealityController